composer.json

```diff
@@ -31,6 +31,7 @@
         "symfony/form": "~2.3|~3.0",
         "symfony/console": "~2.3|~3.0",
         "symfony/phpunit-bridge": "~2.3|~3.0",
+        "symfony/process": "~2.3|~3.0",
```

Comment:
Post processors use ProcessBuilder class from symfony/process component. Maybe it should be included in composer require section?